packages/base64/package.json

```
@@ -33,7 +33,7 @@
   "scripts": {
     "build": "exit 0",
     "prepack": "tsc --build tsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*'",
+    "postpack": "git clean -f '*.d.ts*' tsconfig.build.tsbuildinfo",
```

might as well delete any future ones
```suggestion
    "postpack": "git clean -f '*.d.ts*' *.tsbuildinfo",
```

---

Any thoughts on whether to defer the glob expansion to git?

---

nope

---

I've developed an opinion: https://github.com/Agoric/agoric-sdk/pull/9795/commits/e119569b394e27cdee6718dfafddc04ba90cfaed

hazarded a guess that git might do it a little faster

---

Following your lead. The other effect is that it will match files in subdirectories.